lib/types/sign_command.ex

```diff
@@ -15,5 +20,25 @@ defmodule Kadena.Types.SignCommand do
 
   @impl true
   def new(%SignatureWithHash{} = signature), do: %__MODULE__{command: signature}
-  def new(_sig), do: {:error, :invalid_sign_command}
+
+  def new(args) when is_list(args) do
+    hash = Keyword.get(args, :hash, nil)
+    sig = Keyword.get(args, :sig, nil)
+    pub_key = Keyword.get(args, :pub_key, nil)
```

Comment:
@santiagoboterokommit it is not necessary to define nil as the default value, the `Keyword.get/3` returns it by default.